Firmware/Marlin_main.cpp

```diff
@@ -4093,8 +4092,7 @@ void process_commands()
 		}
 		else if (code_seen_P(PSTR("RESET"))) { // PRUSA RESET
 #if defined(XFLASH) && defined(BOOTAPP)
-            boot_app_magic = BOOT_APP_MAGIC;
-            boot_app_flags = BOOT_APP_FLG_RUN;
```

Comment:
Just wondering, is there any risk with this change?

Reply:
There shouldn't be any risk. It's actually better now. You can look at the bootapp code in the bootloader to check it
https://github.com/prusa3d/stk500v2-prusa/blob/91fc4fa85c8fe001384a6d70359d113abd0325ec/stk500boot.c#L777

If we set the bootapp magic to anything besides 0x55aa55aa. then a watchdog reset will make the bootloader start the firmware immediately. Moreover, BOOT_APP_FLG_RUN is not a thing and it never was a thing.